steps/wip/measure.py

```diff
@@ -0,0 +1,290 @@
+):
+    ansatz_parameters = load_circuit_template_params(ansatz_parameters)
+    # Load qubit op
+    operator = load_qubit_operator(qubit_operator)
+    if isinstance(ansatz_specs, str):
+        ansatz_specs = json.loads(ansatz_specs)
+    if ansatz_specs["function_name"] == "QAOAFarhiAnsatz":
+        ansatz = create_object(ansatz_specs, cost_hamiltonian=operator)
+    else:
+        ansatz = create_object(ansatz_specs)
+
+    if isinstance(backend_specs, str):
+        backend_specs = json.loads(backend_specs)
+    if noise_model is not None:
+        backend_specs["noise_model"] = load_noise_model(noise_model)
+    if device_connectivity is not None:
+        backend_specs["device_connectivity"] = load_circuit_connectivity(
+            device_connectivity
+        )
+
+    backend = create_object(backend_specs)
+
+    if isinstance(cost_function_specs, str):
+        cost_function_specs = json.loads(cost_function_specs)
+
+    if prior_expectation_values is not None:
+        if isinstance(prior_expectation_values, str):
+            prior_expectation_values = load_expectation_values(prior_expectation_values)
+
+    estimator_specs = cost_function_specs.pop("estimator-specs", None)
+    if estimator_specs is not None:
+        if isinstance(estimator_specs, str):
+            estimator_specs = json.loads(estimator_specs)
+        cost_function_specs["estimator"] = create_object(estimator_specs)
+
+    estimation_preprocessors_specs = cost_function_specs.pop(
+        "estimation-tasks-transformations-specs", None
+    )
+    if estimation_preprocessors_specs is not None:
+        cost_function_specs["estimation_preprocessors"] = []
+        for estimation_tasks_transformation_specs in estimation_preprocessors_specs:
+
+            if isinstance(estimation_tasks_transformation_specs, str):
+                estimation_tasks_transformation_specs = json.loads(
+                    estimation_tasks_transformation_specs
+                )
+
+            if prior_expectation_values is not None:
+                # Since we don't know which estimation task transformation uses prior_expectation_values,
+                #    we add it to the kwargs of each one. If not used by a particular transformer, it will be ignored.
+                estimation_tasks_transformation_specs[
+                    "prior_expectation_values"
+                ] = prior_expectation_values
+            cost_function_specs["estimation_preprocessors"].append(
+                create_object(estimation_tasks_transformation_specs)
+            )
+
+    # cost_function.estimator.prior_expectation_values
+    cost_function_specs["target_operator"] = operator
+    cost_function_specs["ansatz"] = ansatz
+    cost_function_specs["backend"] = backend
+    cost_function = create_object(cost_function_specs)
+
+    value_estimate = cost_function(ansatz_parameters)
+
+    save_value_estimate(value_estimate, "value_estimate.json")
+
+
+def hamiltonian_analysis(
```

Comment:
I've deleted this step altogether in #268 .

Reply:
Deleted here: d206a43